recipe/meta.yaml

```diff
@@ -65,3 +65,4 @@ extra:
     - joshmoore
     - jhamman
     - rabernat
+    - dstansby
```

Comment:
@dstansby -- adding you here. Give a 👍 to acknowledge or a comment if you'd like to be left off this roster.